doc/ace-theme-license.txt

```diff
@@ -0,0 +1,42 @@
+MantisBT holds extended license for Ace-Admin theme used in building the Modern UI.
+
+Extended License Terms:
+=======================
+
+This license must be purchased if you intend to license, sublicense, redistribute, or resell the final product.
+
+1. Your use of the item may extend to multiple installations.
+2. You may use the item in work which you are creating for your own purposes or for your clients.
+3. You may license, sublicense, redistribute, or resell the item in the following circumstances:
+	3.1: the item is incorporated into a larger work you have created; or
+	3.2: if you modify the item and resell the end product.
+4. If the item contains licensed components, those components must only be used within the item and you must not extract
+and use them on a stand-alone basis.
+5. If the item was created using materials which are the subject of a GNU General Public License (GPL), your use of the
+item is subject to the terms of the GPL in place of the foregoing conditions (to the extent the GPL applies).
+
+
+*** MantisBT core team is in possession of the proof of purchase for this license ***
+
+
+
+Author affirmation of MantisBT usage requires extended license:
```

Comment:
I would reward this title as: "Author's confirmation that MantisBT usage is covered by the purchased Extended License"